src/renderer/App.tsx

```diff
@@ -55,6 +77,20 @@ function AppContents() {
           open={hasOpenedShortcuts}
           onClose={closeShortcut}
         />
+        <CustomModal
+          open={hasOpenedUpdateTranscriptionAPIKey}
+          onClose={closeUpdateTranscriptionAPIKey}
+        >
+          <CustomModalInner sx={{ width: { xs: 300, sm: 400, lg: 500 } }}>
+            <CloudConfigView
+              prevView={null}
+              closeModal={closeUpdateTranscriptionAPIKey}
+              nextView={null}
+              projectName=""
+              textToDisplay={null}
```

Comment:
The text displayed in the modal should only be the default text if the user has never saved a key before (i.e. the first time using cloud). If they already have provided a key, the modal should look like this:

<img width="504" alt="Screen Shot 2022-08-30 at 10 38 29 AM" src="https://user-images.githubusercontent.com/27614346/187322723-7c41f48f-5700-4bd0-a75b-a4690101a399.png">

Reply:
I'm currently working on this, should be up later today